Documentation/gettingstarted/tls-visibility.rst

```diff
@@ -0,0 +1,330 @@
+even for connections where client to server
+communication is protected by TLS, such as when a client accesses the API service via HTTPS.  This capability is similar to
+what is possible to traditional hardware firewalls, but is implemented entirely in software on the Kubernetes worker node,
+and is policy driven, allowing inspection to target only selected network connectivity.
+
+This type of visibility is
+extremely valuable to be able to monitor how external API services are being used,
+for example, understanding which S3 buckets are being accessed by an given application.
+
+.. include:: gsg_requirements.rst
+
+
+Edit the ClusterRole for Cilium to give it access to Kubernetes secrets
+
+
+.. parsed-literal::
+
+    $ kubectl edit clusterrole cilium -n kube-system
+
+Add the following section at the end of the file:
+
+
+.. parsed-literal::
+
+  - apiGroups:
+    - ""
+    resources:
+    - secrets
+    verbs:
+    - get
+
+
+Deploy the Demo Application
+===========================
+
+To demonstrate TLS-interception we will use the same ``mediabot`` application that we used for the DNS-aware policy example.
+This application will access the Star Wars API service using HTTPS, which would normally mean that network-layer mechanisms
+like Cilium would not be able to see the HTTP-layer details of the communication, since all application data is encrypted
+using TLS before that data is sent on the network.
+
+In this guide we will learn about:
+
+- Creating an internal Certificate Authority (CA) and associated certificates signed by that CA to enable TLS interception.
+- Using Cilium network policy to select the traffic to intercept using DNS-based policy rules.
+- Inspecting the details of the HTTP request using cilium monitor (accessing this visibility data via Hubble, and applying
+  Cilium network policies to filter/modify the HTTP request is also possible, but is beyond the scope of this simple Getting Started Guide)
+
+
+First off, we will create a single pod ``mediabot`` application:
+
+.. parsed-literal::
+
+    $ kubectl create -f \ |SCM_WEB|\/examples/kubernetes-dns/dns-sw-app.yaml
+    $ kubectl get po
+    NAME                             READY   STATUS    RESTARTS   AGE
+    pod/mediabot                     1/1     Running   0          14s
+
+
+A Brief Overview of the TLS Certificate Model
+=============================================
+
+TLS is a protocol that "wraps" other protocols like HTTP and ensures that communication between client and
+server has confidentiality (no one can read the data except the intended recipient), integrity (recipient
+can confirm that the data has not been modified in transit), and authentication (sender can confirm that
+it is talking with the intended destination, not an impostor).  We will provide a highly simplified overview
+of TLS in this document, but for full details, please see XXX .
```

Comment:
` please see XXX .`

Reply:
thanks for catching that!   Just pushed a fix with link inserted.